huaweicloud/services/iec/resource_huaweicloud_iec_server.go

```diff
@@ -0,0 +1,593 @@
+package iec
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/ecs/v1/cloudservers"
+	"github.com/chnsz/golangsdk/openstack/iec/v1/cloudvolumes"
+	"github.com/chnsz/golangsdk/openstack/iec/v1/common"
```

Comment:
it's better to use `ieccommon` as the alias of iev/v1/common, and keep `huaweicloud/common` as it is.

Reply:
done